File: laboratorio-1-respuestas/node/main.js

function main() {
    console.log("Laboratorio 1");
    const result = sum(2,2);
     console.log("El resultado de la suma es "+ result);
     const resultWithMe = dependWithMe(3,2);
     console.log("El resultado de la operacion es "+ resultWithMe);
}
function sum(x,y){
    return x+y;
}

function dependWithMe(x, y){
    if (x > y) return x * y;
    if (x < y) return (x * y)-(x + y);
    return (x * y)*2;
}
main();

File: laboratorio-1-respuestas/java/Main.java

// javac Main.java
// java Main

public class Main {

    public static void main(String [] argv){
        System.out.println("Laboratorio 1");
        int result = sum(2,2);
        int resultWithMe = dependWithMe(2,2);
        System.out.println("La suma es "+ result);
        System.out.println("La operacion es "+ resultWithMe);
    }

    public static  int sum(int x, int y){
        return  x+y;
    }
    public static int dependWithMe(int x, int y) {
        if (x > y) {
            return x * y;
        }
        if (x < y) {
            return (x * y) - (x + y);
        }
        return (x * y) * 2;
    }
}